src/main/kotlin/gg/essential/elementa/markdown/drawables/ParagraphDrawable.kt

```diff
@@ -238,8 +234,13 @@ class ParagraphDrawable(
 
         // We can have extra drawables in the current line that didn't get handled
         // by the last iteration of the loop
-        if (currentLine.isNotEmpty())
+        if (currentLine.isNotEmpty()) {
             lines.add(currentLine.toList())
+            currY += maxLineHeight
+        } else {
+            // There isn't a next line, so this space shouldn't be there
+            currY -= config.paragraphConfig.spaceBetweenLines
```

Comment:
Not sure if it's possible in practice but if a paragraph doesn't have any lines, then we never call `gotoNextLine` and so in that case we shouldn't subtract `spaceBetweenLines`.